server/views/applications/pages/oasys-import/offence-details.njk

```diff
@@ -0,0 +1,50 @@
+{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
+{% from "../../components/riskWidgets/macro.njk" import widgets %}
+
+{% extends "../layout.njk" %}
+
+{% set columnClasses = "govuk-grid-column-full" %}
+
+{% block questions %}
+  <div class="govuk-grid-row">
+    <div>
+      <h1 class="govuk-heading-l">{{page.title}}</h1>
+      <p>Imported from OASys 23 October 2022</p>
+    </div>
+
+    {{ mojSubNavigation({
+        label: 'Sub navigation',
+        attributes: {
+          'role': 'tablist'
+        },
+        items: [{
+          text: 'RoSH summary',
+          href: paths.applications.pages.show({ id: applicationId, task: 'oasys-import', page: 'rosh-summary' })
+        }, 
+        {
+          text: 'Offence details',
+          active: true
+        }, 
+        {
+          text: 'Supporting information'
+        },  
+        {
+          text: 'Risk management plan'
+        },
+        {
+          text: 'Risk to self'
+        }]
+      }) }}
+  </div>
+
+  <div class="govuk-grid-row">
```

Comment:
Could we DRY these templates up a bit too?

Reply:
Same here, will do it later. Makes sense to do a macro I think